src/linux/mod.rs

```diff
@@ -438,14 +440,15 @@ impl GeneralReadout for LinuxGeneralReadout {
         }
 
         let terminal = terminal_name();
+        let terminal = terminal_replacements(&terminal);
```

Comment:
I think this should be a one-liner, the function should also be renamed to convey its function more clearly.